libs/sdk-ui-kit/src/@ui/UiButton/UiButton.tsx

```diff
@@ -35,11 +36,16 @@ export interface UiButtonProps {
     tooltip?: React.ReactNode;
     onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
     dataId?: string;
+    dataTestId?: string;
     accessibilityConfig?: IUiButtonAccessibilityConfig;
 }
 
 const { b, e } = bem("gd-ui-kit-button");
 
+const getGeneratedTestId = (label: string, ariaLabel: string) => {
```

Comment:
The second argument should be either optional or `string | undefined` imho. Please, change it in the next PR you will do.